samples/annotate/pom.xml

```diff
@@ -61,7 +61,12 @@
 		</dependency>
 		<dependency>
 			<groupId>com.sun.xml.bind</groupId>
-			<artifactId>jaxb-impl</artifactId>
+			<artifactId>jaxb-core</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
We could use `log4j-jcl` adapter instead of `commons-logging`, would it be better?